xtevent/xtevent.sthlp

```diff
@@ -164,17 +164,14 @@ be used as an instrument.
 {it:policyvar_imputed}. The following imputation types can be implemented:
 
 {phang2}
-{cmd:impute(nuchange)} imputes missing values in {it:policyvar} according to {it:no-unobserved change}: It assumes that, 
+{cmd:impute(nuchange)} imputes missing values in {it:policyvar} according to {it:no-unobserved change}: it assumes that, 
 for each unit: i) in periods before the first observed value, the policy value is the same as the first observed value; and
  ii) in periods after the last observed value, the policy value is the same as the last observed value.
 
 {phang2}
 {cmd:impute(stag)} applies {it:no-unobserved change} if {it:policyvar} satisfies staggered-adoption assumptions for all units: 
 i) {it:policyvar} must be binary; and ii) once {it:policyvar} reaches the adopted-policy state, it never reverts to the 
-unadopted-policy state. See Freyaldenhoven et al. (2019) for detailed explanation of the staggered case. Additionally in 
-the {it:policyvar}, for each unit: i) the first-observed value must be the unadopted-policy-state value, and the last-observed
-value must be the adopted-policy-state value; or ii) all policy values in the observed data range must be either 
-adopted-policy-state values or unadopted-policy-state values.  
+unadopted-policy state. See Freyaldenhoven et al. (2019) for detailed explanation of the staggered case.

```

Comment:
We had let case ii) because although it didn't seem to fit in the staggered description above, we didn't want the program to fail the staggered-adoption check if there were never-treated or always-treated units.